pyvista/utilities/fileio.py

```diff
@@ -233,3 +251,57 @@ def read_exodus(filename,
 
     reader.Update()
     return pyvista.wrap(reader.GetOutput())
+
+
+def _read_meshio(filename, file_format = None):
```

Comment:
Can you make this a public method much like the `read_exodus` method incase a user or downstream project wants to use just this routine for file reading

Reply:
i.e. no leading underscore

Reply:
Done!